actions/ckeditor/upload.php

```diff
@@ -16,8 +16,8 @@
 
 $user = elgg_get_logged_in_user_entity();
 
-$max_width = (int) elgg_get_plugin_setting('upload_max_width', 'ckeditor_addons', 600);
-$resized = get_resized_image_from_existing_file($_FILES['upload']['tmp_name'], $max_width, $max_width);
+$resized = get_uploaded_file('upload');
```

Comment:
I do not want resize function, because somebody will highlight the whole image to view the detail,  especially if somebody uploaded a image including many small letters, if you resized it ,it would be blur